app/src/main/java/de/stephanlindauer/criticalmaps/fragments/TwitterFragment.java

```diff
@@ -82,6 +100,12 @@ public void onActivityCreated(final Bundle savedState) {
         new TwitterGetHandler(this).execute();
     }
 
+    @Override
+    public void onCreateOptionsMenu(@NonNull Menu menu, @NonNull MenuInflater inflater) {
+        super.onCreateOptionsMenu(menu, inflater);
+        menu.clear();
```

Comment:
I agree that having the menu on all screens is probably something we should change. But first we nee to find a better to clearly communicate how to stop tracking (aka "closing" the app).
But removing it just here is inconsistent. So i would rather keep the menu for now.

Reply:
I agree